dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/MetadataDumper.java

```diff
@@ -292,9 +293,32 @@ public <T> T runChildTask(Task<T> task) throws MetadataDumperUsageException {
                         + "* Metadata has been saved to " + outputFile + "\n"
                         + "**********************************************************"
                 );
+            } else if (connector instanceof AbstractSummaryHolderJdbcConnector) {
+                AbstractSummaryHolderJdbcConnector.Summary summary =
+                    ((AbstractSummaryHolderJdbcConnector) connector).getSummary();
+                long totalRows = summary.rowCount();
+                if (totalRows > 0) {
+                    System.out.println(
+                        "**********************************************************\n"
+                            + "* Logs has been saved to " + outputFile + "\n"
+                            + "* Earliest Row Found at " + summary.earliestInterval().getStart()
+                            + "\n"
+                            + "* Latest Row Found at " + summary.latestInterval().getEndInclusive()
+                            + "\n"
+                            + "* Total Number of Rows Dumped " + totalRows + "\n"
```

Comment:
nit: In my opinion, the capitalization of first letters is unnecessary. I would suggest to change to `Total number of rows dumped`.